docs/index.rst

```diff
@@ -1,5 +1,14 @@
 .. include:: README.rst
 
+.. note::
+
+   Because the Bigquery client uses the third-party :mod:`requests` library
+   by default and the Bigquery-Storage client uses :mod:`grpcio` library,
```

Comment:
I think it should (must?) be spelled BigQuery (camel case), as this is an official name, but I might be wrong.

Reply:
Yes I have seen `BigQuery` in camel case in doc's other pages so changed it.

Reply:
Thanks!